packages/discord.js/typings/index.d.ts

```diff
@@ -3277,6 +3254,37 @@ export class Webhook extends WebhookMixin() {
   public send(options: string | MessagePayload | WebhookMessageCreateOptions): Promise<Message>;
 }
 
+export class IncomingWebhook extends BaseWebhook {
+  public type: WebhookType.Incoming;
+  public applicationId: Snowflake | null;
+  public sourceGuild: null;
+  public sourceChannel: null;
+  public token: string;
+}
+
+export class UserCreatedWebhook extends IncomingWebhook {
+  public applicationId: null;
+  public owner: User | APIUser;
+}
+
+export class ApplicationCreatedWebhook extends BaseWebhook {
+  public type: WebhookType.Application;
+  public applicationId: Snowflake;
+  public sourceGuild: null;
+  public sourceChannel: null;
+  public token: string;
+}
+
+export class ChannelFollowerWebhook extends BaseWebhook {
+  public type: WebhookType.ChannelFollower;
+  public applicationId: null;
+  public sourceGuild: Guild | APIPartialGuild;
+  public sourceChannel: NewsChannel | APIPartialChannel;
+  public token: null;
+}
```

Comment:
This is also a similar issue. None of these classes are exported but as far as TS is concerned they are exported. Which is now more of a problem than before as someone may do

```ts
import { ChannelFollowerWebhook } from 'discord.js';

const webhook = [they got a webhook from somewhere];

if (webhook instanceof ChannelFollowerWebhook) {
  // do something
}
```
and their code will make a runtime error because you can't instanceof undefined